docs/UI/OnLanguageChange/assets/OnLanguageChange.txt

```diff
@@ -2,22 +2,24 @@ participant App
 participant SDL
 participant HMI
 
-note over App,HMI:<align:center> **//Pre-conditions://**\n1. App registered with **EN-US** language\n2. Current HMI display language is **EN-US**
+note over App,HMI:<align:center> **//Pre-conditions://**\n1. `appID` is registered with **`EN-US`** language\n2. Current HMI UI language is **`EN-US`**
 
-note over HMI:<align:center>User switched\nHMI display language\nto **<FR-CA>**
+note over HMI:<align:center>User switched\nlanguage to **`FR-CA`**
 
-HMI->>SDL:UI.OnLanguageChange (<**FR-CA**>)
+HMI->>SDL:UI.OnLanguageChange (**`FR-CA`**)
 SDL->>SDL:<align:center><align:center>Overwrite current language\nin `HMICapabilitiesCacheFile`
```

Comment:
```suggestion
SDL->>SDL:<align:center><align:center>Overwrite current hmiDisplayLanguage\nin `HMICapabilitiesCacheFile`
```
Wording still needs to be fix

Reply:
@jacobkeeler please see updates in ba730b1
If the wording is not applicable, please clarify the required changes.